sensor.py

```diff
@@ -97,10 +97,12 @@ def device_class(self):
         """Return the device class for this sensor."""
         return self._device_class
 
-    def _update_callback(self, _device):
+    def _update_callback(self, device):
         """Run when invoked by pyvantage when the device state changes."""
         self.schedule_update_ha_state()
 
+        if self._vantage_device.kind == "button":
```

Comment:
We talked about maybe having a config directive fire_button_events, but unless there's a precedent in other modules for guarding firing of events with a config setting, they're almost certainly lightweight enough and rare enough to just let them be fired and have them be ignored.

Reply:
That was my gut feeling.  Happy to put a config option in if you think it matters, but otherwise I'll just leave it.